packages/ts-client/src/webrtc/tracks/LocalTrackManager.ts

```diff
@@ -100,20 +100,18 @@ export class LocalTrackManager<EndpointMetadata, TrackMetadata> {
   };
 
   public replaceTrackHandler = async (
-    // todo remove webrtc with newTrackMetadata
     webrtc: WebRTCEndpoint<EndpointMetadata, TrackMetadata>,
     trackId: string,
     newTrack: MediaStreamTrack | null,
-    // todo remove webrtc with newTrackMetadata
-    newTrackMetadata?: TrackMetadata,
   ): Promise<void> => {
     this.ongoingTrackReplacement = true;
     try {
-      await this.local.replaceTrack(webrtc, trackId, newTrack, newTrackMetadata);
+      await this.local.replaceTrack(webrtc, trackId, newTrack);
     } catch (_error) {
+      /* empty */
```

Comment:
Why does `local.replaceTrack` throw exception, just to be ignored by `replaceTrackHandler`?